### ISSUE TITLE
New shared preview keywords have same pixelScale for all preview sizes

### ISSUE THREAD

**Issue by grahambell** [Starlink/ORAC-DR#8]
Now that the keywords are generated once for all sizes of a preview, the pixelScale can only be right for one of them.

```
$ exiftool jcmt_20140918-8d9f162ee524dc4e944ad9aa708e11ae_reduced-850um_preview_64.png
...
Keywords                        : [...] astro:pixelScale=19.7105522490712, [...]
...
$ exiftool jcmt_20140918-8d9f162ee524dc4e944ad9aa708e11ae_reduced-850um_preview_1024.png
...
Keywords                        : [...] astro:pixelScale=19.7105522490712, [...]
...
```

**Comment by sfgraves** [comment]
Do we still care about the keywords in the preview pngs? They must be wrong for the spectral images we send to CADC anyway, seeing as we stick two pngs together.

**Comment by timj** [comment]
I'd obviously prefer if they were right when they could be right rather than abandoning the whole concept of doing it properly.

**Comment by sfgraves** [comment]
Actually, I was wrong and the CADC-served ACSIS preview images don't have any incorrect keywords although they also don't have many useful keywords in them. They currently look like:

![Example HARP preview image](http://www.cadc-ccda.hia-iha.nrc-cnrc.gc.ca/AdvancedSearch/preview/JCMT/jcmt_acsis_00074_20071221T145444_reduced-345796MHz-1000MHzx2048-1_preview_1024.png)

(... although usually without the hole in the middle of the source...)

And the only astro keyword the ACSIS preview included is astro:name, so I was wrong about these. (I suspect it just inherits from the first image when jsawrapdr sticks the images together).

I'll take a look at fixing the pixelScale for the maps, hopefully this week.

Does anyone know if we would have a summary of the use case/intended behaviour for the keywords anywhere, maybe in ORAC-DR? (Or, @timj , if you happen to remember what they were intended to be used for?) It would be nice to be sure we don't break anything if we ever change up the images. There  was discussion in the past about including coordinate axes in the images, and I remember one of the reasons against it was breaking the image keywords if there was non-map area in the image.

**Comment by timj** [comment]
I was aiming for compatibility with Microsoft World Wide Telescope and the Flickr astronomy feeds. The standard we are using is also used by HST. I think it's AVM: http://www.virtualastronomy.org/avm_metadata.php

**Comment by grahambell** [comment]
Hopefully this is fixed in commit cc6406f53f5455365c1c42f5a6a0e976b6cd0417.  I think astro:pixelScale is the only one of the headers which needs to change depending on the resolution, so it is now recalculated for each resolution while the others are still only prepared once.
